charts/redis/templates/service.yaml

```
@@ -14,6 +14,18 @@ metadata:
   {{- end }}
 spec:
   type: {{ .Values.service.type }}
+  {{- if and .Values.service.loadBalancerIP (eq .Values.service.type "LoadBalancer") }}
+  loadBalancerIP: {{ .Values.service.loadBalancerIP }}
+  {{- end }}
+  {{- if and (eq .Values.service.type "LoadBalancer") .Values.service.loadBalancerSourceRanges }}
+  loadBalancerSourceRanges:
+  {{ with .Values.service.loadBalancerSourceRanges }}
+{{ toYaml . | indent 4 }}
+{{- end }}
+  {{- end }}
```

```suggestion
    {{- with .Values.service.loadBalancerSourceRanges }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
    {{- end }}
```

Also, can you ensure the indentation is consistent, so that the template remains readable?